packages/react-native-gesture-handler/src/web/handlers/GestureHandler.ts

```diff
@@ -678,29 +688,58 @@ export default abstract class GestureHandler implements IGestureHandler {
   // Handling config
   //
 
-  public updateGestureConfig({
-    enabled = true,
-    dispatchesAnimatedEvents = false,
-    ...props
-  }: Config): void {
-    this._config = {
-      enabled,
-      dispatchesAnimatedEvents,
-      ...props,
-    };
+  public setGestureConfig(config: Config) {
+    this.resetConfig();
+    this.updateGestureConfig(config);
+  }
+
+  public updateGestureConfig(config: Config): void {
+    if (config.enabled !== undefined && this.enabled !== config.enabled) {
+      this.enabled = config.enabled;
+      this.delegate.onEnabledChange(this.enabled);
+    }
+
+    if (config.hitSlop !== undefined) {
+      this.hitSlop = config.hitSlop;
+      this.hitSlop = config.hitSlop;
+      this.validateHitSlops();
+    }
+
+    if (config.dispatchesAnimatedEvents !== undefined) {
+      this.forAnimated = config.dispatchesAnimatedEvents;
```

Comment:
```suggestion
      this.dispatchesAnimatedEvents = config.dispatchesAnimatedEvents;
```

Can we align those names?